Objects/typeobject.c

```diff
@@ -252,8 +252,9 @@ _PyType_InitCache(PyInterpreterState *interp)
 
 
 static unsigned int
-_PyType_ClearCache(struct type_cache *cache)
+_PyType_ClearCache(PyInterpreterState *interp)
 {
+    struct type_cache *cache = get_type_cache();
```

Comment:
I prefer to avoid this function which reads again interp, whereas you already have it. Just use :
```suggestion
    struct type_cache *cache = &interp->type_cache;
```